roles/idrac_os_deployment/tasks/iso/extract_iso.yml

```
@@ -5,6 +5,15 @@
   changed_when: idrac_os_deployment_extract_cmd_out.rc == 0
   failed_when: idrac_os_deployment_extract_cmd_out.rc != 0
   ansible.builtin.command: "{{ idrac_os_deployment_xorriso_cmd | format(idrac_os_deployment_iso_file, idrac_os_deployment_iso_extract_dir) }}"
+  when: "'ubuntu' not in idrac_os_deployment_iso_file.lower()"
```

This would break if the iso file name is not ubuntu, hence good to check with the `os_name` input.
```suggestion
  when: os_name.lower() in ['rhel', 'esxi']
```

---

Noted, will change it